ui/src/components/app_bar/notification/Notification.vue

```diff
@@ -9,7 +9,7 @@
         :content="showNumberNotifications"
         :value="showNumberNotifications"
         overlap
-        offset-x="20"
+        offset-x="30"
```

Comment:
Don't you think it's better to set this value according to the amount of notifications?